client/app/app.scss

```diff
@@ -40,6 +40,15 @@ h2, h3, h4, h5, h6 {
   width: 300px;
 }
 
+.wallet {
```

Comment:
This code should be in the wallet module css file, not inside the main application css.